.github/workflows/build.yml

```diff
@@ -26,7 +26,7 @@ jobs:
       - run: yarn audit
 
       - run: yarn lint
-      - run: yarn workspace @itwin/itwinui-react lint:copyright ${{ github.workspace }}/*/**.{js,ts,tsx}
+      - run: yarn lint:copyright ${{ github.workspace }}/*/**.{js,ts,tsx}
```

Comment:
Looks like this is broken (even before this PR). It's not checking all files
![image](https://user-images.githubusercontent.com/9084735/168167764-0c1d9aba-43ab-4dbf-9dcf-88ee258265ed.png)

Reply:
Adding quotes doesn't help
![image](https://user-images.githubusercontent.com/9084735/168168331-717f155a-bbad-401b-949f-6e6550031e26.png)

Reply:
So is it working?

Reply:
Works after most recent change. I'm hardcoding the pattern inside `copyrightLinter.js`

Tested failure and success cases in CI 👍
https://github.com/iTwin/iTwinUI-react/runs/6476456067
https://github.com/iTwin/iTwinUI-react/runs/6476485106